doit/dependency.py

```diff
@@ -409,21 +475,16 @@ def save_success(self, task):
                 self._set(task.name, "result:", get_md5(task.result))
 
         # file-dep
+        self._set(task.name, 'checker:', self.checker.__class__.__name__)
```

Comment:
I didn't looked at the db stuff, so it must be possible but if the checker changes from timestamp to the default one, the 'checker:' column will exists so what must be done then ? Is there a way to delete a column in the db backend ? So, is it really a useful optimization to not save this value for the default checker ?